packages/ai/src/ai/web/scheduler/scheduler.py

```
@@ -0,0 +1,304 @@
+            replace_existing=False,
+        )
+
+        logger.info('Schedule added: id=%s pipeline=%s cron=%s', schedule_id, pipeline_id, cron_expression)
+        return schedule_id
+
+    def remove_schedule(self, schedule_id: str) -> bool:
+        """Remove a schedule by ID. Returns True if removed, False if not found."""
+        try:
+            self._scheduler.remove_job(schedule_id)
+            logger.info('Schedule removed: id=%s', schedule_id)
+            return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)